src/schema/v2/fair.ts

```diff
@@ -288,6 +285,11 @@ export const FairType = new GraphQLObjectType<any, ResolverContext>({
         },
       },
       startAt: date,
+      summary: {
+        type: GraphQLString,
```

Comment:
Unclear to me whether this ever will be markdown... I don't _think_ so. Easy enough to change later if we learn differently.